### PR TITLE
feat: multichain activity history

### PR DESCRIPTION
**Pull request by cartcrom**
## Description

Updates activity query to request multi-chain results

_Linear ticket:_ https://linear.app/uniswap/issue/WEB-3160/add-multichain-gql-activity-history

| Before | After |
| ------ | ----- |
| image | image |

### CODE REVIEW

File: src/graphql/data/util.tsx [hunk 1]
```diff
@@ -201,7 +201,7 @@ export const BACKEND_SUPPORTED_CHAINS = [
   Chain.Bnb,
   Chain.Celo,
 ] as const
-export const BACKEND_NOT_YET_SUPPORTED_CHAIN_IDS = [ChainId.AVALANCHE] as const
+export const BACKEND_NOT_YET_SUPPORTED_CHAINS = [ChainId.AVALANCHE] as const
```
Comment (cbachmeier) [inline src/graphql/data/util.tsx:204]: why was this renamed? In my mind `chains` refers to the BE chain name strings where as chainId is specifically the number representation

File: src/graphql/data/activity.graphql [hunk 2]
```diff
@@ -161,10 +161,10 @@ fragment AssetActivityParts on AssetActivity {
 }
 
 # TODO(UniswapX): return to a pagesize of 50 pre-launch
-query Activity($account: String!) {
-  portfolios(ownerAddresses: [$account]) {
+query Activity($account: String!, $chains: [Chain!]!) {
```
Comment (cbachmeier) [inline src/graphql/data/activity.graphql:164]: non-blocking not: but can we make this optional?

**Review by cbachmeier — APPROVED** [review]
LGTM, non-blocking nits

File: src/graphql/data/activity.graphql (hunk 2 above)
Comment (cartcrom) [reply src/graphql/data/activity.graphql:164]: If BE defaulted to returning all chains I think that would be best, but since it doesn't, I think it's best to require it so that callers don't accidentally request mainnet-only data without realizing

**Review by just-toby — APPROVED** [review]

**Comment by cartcrom** [comment]
merged into new repo instead
